tests/Behat/features/element-editor.feature

```diff
@@ -121,13 +116,25 @@ Feature: View types of elements in an area on a page
     And I wait for 1 seconds
     Then I should see a ".tox-tinymce" element
 
-    # More actions menu can be accessed with keyboard actions
+    # More actions menu can be accessed and hidden with keyboard actions
+    # Doing so does not collapse the block as a whole
     When I press the "tab" key globally
     Then I should not see "Duplicate"
+    And I should see a ".tox-tinymce" element
+    # open menu with space and close with escape
     When I press the "space" key globally
     Then I should see "Duplicate"
+    And I should see a ".tox-tinymce" element
+    When I press the "escape" key globally
```

Comment:
An update to bootstrap and/or reactstrap has changed the behaviour to give focus to the first element in the dropdown when opening with the keyboard, so we need to use escape to close it rather than hitting space/enter again.